apps/chat/src/components/Chat/Publish/PublicationChatControls.tsx

```diff
@@ -64,6 +64,11 @@ export function PublicationControlsView<
         isPreview: false,
       }),
     );
+    dispatch(
+      ConversationsActions.selectConversations({
+        conversationIds: [],
+      }),
+    );
```

Comment:
I think this code will break the combined request review

Reply:
It might not! it's handler for `Back to publication request`. To see publication in any cases no selected conversation should be